lib/octokit/client/repositories.rb

```diff
@@ -192,6 +194,23 @@ def transfer_repository(repo, new_owner, options = {})
       end
       alias :transfer_repo :transfer_repository
 
+      # Clone a template repository for a user or organization
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub template repository
+      # @param name [String] Name of the new repo
+      # @option options [String] :owner Organization or user who will own the new repository.
+      # @option options [String] :description Description of the repo
+      # @option options [String] :private `true` makes the repository private, and `false` makes it public.
+      # @return [Sawyer::Resource] Repository info for the new repository
+      def clone_template_repository(repo, name, options = {})
+        options.merge! :name => name
+        options = ensure_api_media_type(:template_repositories, options)
+        post "#{Repository.path repo}/generate", options
+      end
+      alias :create_repository_from_template :clone_template_repository
+      alias :clone_template_repo :clone_template_repository
+      alias :create_repo_from_template :clone_template_repository
```

Comment:
There are a lot of aliases here, could we just keep `clone_template_repo` ?

Reply:
Done, I've actually changed the name completely to `create_github_repository_from_template` with one alias: `create_github_repo_from_template`